docs/getting-started.md

```diff
@@ -0,0 +1,105 @@
+# Getting Started - BAG Bevragen API
+De 'BAG Bevragen' API levert actuele gegevens uit de Basisregistratie Adressen en Gebouwen. Dit betekent dat geen historische gegevens kunnen worden gevraagd en dat gegevens met een eindstatus niet worden geleverd.
+
+De API is gespecificeerd met behulp van de OpenAPI Specifications (OAS3).
+
+Om aan te sluiten kun je de volgende stappen doorlopen:
+1. [Meld je aan bij het kadaster om toegang te krijgen](#Aanmelden-om-aan-te-sluiten)
+2. [Bekijk de functionaliteit en specificaties](#Functionaliteit)
+3. [Probeer en test de API](#Probeer-en-test-de-API)
+
+## Aanmelden om aan te sluiten
+Vraag [een API-key aan](https://formulieren.kadaster.nl/aanvragen_bag_api_key_testomgeving) om toegang te krijgen tot de BAG API.
```

Comment:
Voor de productie omgeving is de link:
https://formulieren.kadaster.nl/aanvragen_bag_api_key_productieomgeving.